docs/configuration/cosmos-conf.rst

```diff
@@ -0,0 +1,56 @@
+Cosmos Config
+=============
+
+This page contains the list of all available Airflow configurations and environment variables specific to ``astronomer-cosmos`` that can be set in the airflow.cfg file or using environment variables.
+
+.. note::
+    For more information, see `Setting Configuration Options <https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html>`_.
+
+**Sections:**
+
+- [cosmos]
+- [openlineage]
+
+[cosmos]
+~~~~~~~~
+
+1. ``cache_dir``:
+    The directory used for caching Cosmos data.
+
+    - Default: ``{TMPDIR}/cosmos_cache`` (where ``{TMPDIR}`` is the system temporary directory)
+
+2. ``enable_cache``:
+    Enable or disable caching of Cosmos data.
+
+    - Default: ``True``
+
+3. ``propagate_logs``:
+    Whether to propagate logs in the Cosmos module.
+
+    - Default: ``True``
+
+4. ``dbt_docs_dir``:
+    The directory path for dbt documentation.
+
+    - Default: ``None``
+
+5. ``dbt_docs_conn_id``:
+    The connection ID for dbt documentation.
+
+    - Default: ``None``
+
+[openlineage]
+~~~~~~~~~~~~~
+
```

Comment:
I think it's worth adding a reference that this is the OpenLineage provider configuration, and where users can 
https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/guides/user.html

Reply:
I have added a note on this.